config/locales/en.yml

```diff
@@ -43,11 +43,16 @@ en:
               taken: "Username has already been taken"
 
   app:
+    already_have_account: "Already have account?"
+    do_not_have_account_yet: "Don't have account yet?"
```

Comment:
just minor thing, should it be `Already have account?` & `Don't have an account yet?`

Reply:
Sorry I don't get it, What is the difference between my code and your comment?

Reply:
sorry @gutakk. I mistyped it. I mean adding "an" to the sentences.
"Already have account?" -> ""Already have an account?"
"Don't have account yet?" -> "Don't have an account yet?"

Reply:
Fixed in 2aeda79beaa4edd0b886fa97133f4ba41f67da9f